openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClasses.java

```diff
@@ -275,4 +289,20 @@ public void tearDown(StfCoreExtension test, StfSharedClassesExtension sharedClas
 		sharedClasses.doDestroySpecificCache("Destroy cache", "-Xshareclasses:name=" + SCSoftmxTestUtil.CACHE_NAME + ",cacheDir=" + cacheDir + "${cacheOperation}", SCSoftmxTestUtil.CACHE_NAME, cacheDir);
 		sharedClasses.doDestroySpecificNonPersistentCache("Destroy cache", "-Xshareclasses:name=" + SCSoftmxTestUtil.CACHE_NAME + ",cacheDir=" + cacheDir + "${cacheOperation}", SCSoftmxTestUtil.CACHE_NAME, cacheDir);
 	}
+	public void removeAllContentsAndDeleteDir(Path directory) throws IOException {
+		System.out.println("Attempting to delete directory and its contents: " + directory.toAbsolutePath());
+		try (Stream<Path> paths = Files.walk(directory)) {
+			paths.sorted(Comparator.reverseOrder())
```

Comment:
Please just use simple logic to delete files. We do not need `paths.sorted`. And having a try without a catch is not correct.

https://stackoverflow.com/questions/20281835/how-to-delete-a-folder-with-files-using-java